src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayilyPay = 40;
+  const discountSmall = 20;
+  const discountBig = 50;
+  const totalPrice = days * dayilyPay;
+
+  if (days > 2 && days < 7) {
```

Comment:
```suggestion
  if (days >= 3 && days < 7) {
```
It's ok but better to stick with numbers that describe the number of days more precisely.
It's not critical but good practice